src/privacy/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<title>Privacy Policy | Caselaw Access Project</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="robots" content="noindex" />
+		<meta name="author" content="Your name" />
+		<meta name="description" content="Caselaw Access Project Privacy Policy" />
+		<meta property="og:title" content="Caselaw Access Project" />
+		<meta property="og:description" content="Caselaw Access Project Privacy Policy" />
+		<meta property="og:url" content="https://case.law/" />
+		<meta property="og:type" content="article" />
+		<meta property="og:site_name" content="Caselaw Access Projects" />
```

Comment:
Would you be willing to update these meta tags to something more like https://github.com/harvard-lil/capstone-static/blob/main/src/about/index.html#L3-L34?

We did have meta tags like the above originally, but they were just placeholders.

If not, we can make a separate ticket for fixing.